packages/react-components/src/atoms/Button.tsx

```diff
@@ -10,7 +10,6 @@ const linkStyles = css({
   display: 'inline',
   '> svg': {
     minHeight: '1em',
-    height: '100%',
```

Comment:
This does not explain everything, right? Why is the parent growing??

Reply:
The parent without this. Its like padding is added each time its resized and then the SVG takes 100%